texttospeech/synthesize_text/synthesize_text.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+/*
+Example usage:
+	go get -u cloud.google.com/go/texttospeech/apiv1
+	go run synthesize_text.go --text "hello"
+	go run synthesize_text.go --ssml "<speak>Hello there.</speak>"
+*/
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"golang.org/x/net/context"
+
+	texttospeech "cloud.google.com/go/texttospeech/apiv1"
+	texttospeechpb "google.golang.org/genproto/googleapis/cloud/texttospeech/v1"
+)
+
+// [START tts_synthesize_text]
+func SynthesizeText(text, output_file string) error {
+	ctx := context.Background()
+
+	client, err := texttospeech.NewClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	input := texttospeechpb.SynthesisInput{
+		InputSource: &texttospeechpb.SynthesisInput_Text{text}}
+
+	// Note: the voice can also be specified by name.
+	// Names of voices can be retrieved with client.ListVoices().
+	voice := texttospeechpb.VoiceSelectionParams{
+		LanguageCode: "en-US",
+		SsmlGender: texttospeechpb.SsmlVoiceGender_FEMALE}
+
+	audioConfig := texttospeechpb.AudioConfig{
+		AudioEncoding: texttospeechpb.AudioEncoding_MP3}
+
+	request := texttospeechpb.SynthesizeSpeechRequest{
+		Input: &input,
+		Voice: &voice,
+		AudioConfig: &audioConfig,
+	}
+	response, err := client.SynthesizeSpeech(ctx, &request)
+	if err != nil {
+		return err
+	}
+
+	err = ioutil.WriteFile(output_file, response.AudioContent, 0644)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Audio content written to file: %v\n", output_file)
+	return nil
+}
+// [END tts_synthesize_text]
+
+// [START tts_synthesize_ssml]
+func SynthesizeSsml(ssml, output_file string) error {
+	ctx := context.Background()
+
+	client, err := texttospeech.NewClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	// Note: ssml must be well-formed according to:
+	//       https://www.w3.org/TR/speech-synthesis/
+	// Example: <speak>Hello there.</speak>
+	input := texttospeechpb.SynthesisInput{
+		InputSource: &texttospeechpb.SynthesisInput_Ssml{ssml}}
+
+	// Note: the voice can also be specified by name.
+	// Names of voices can be retrieved with client.ListVoices().
+	voice := texttospeechpb.VoiceSelectionParams{
+		LanguageCode: "en-US",
+		SsmlGender: texttospeechpb.SsmlVoiceGender_FEMALE}
+
+	audioConfig := texttospeechpb.AudioConfig{
+		AudioEncoding: texttospeechpb.AudioEncoding_MP3}
```

Comment:
The style we use in this repo is to have the closing `}` on a newline. Can you change that everywhere?

Reply:
Sure, changed